examples/spotify_example.py

```diff
@@ -96,25 +105,30 @@
     print("Failed to launch spotify controller due to credential error")
     sys.exit(1)
 
-# Query spotify for active devices
-devices_available = client.devices()
-
-# Match active spotify devices with the spotify controller's device id
-for device in devices_available["devices"]:
-    if device["id"] == sp.device:
-        spotify_device_id = device["id"]
-        break
+# The chromecast device does not show up as part of the public API get devices
+# call until it starts playing. The only way to do so is to transfer playback
+# by call this endpoint that's not part of their public API.
+headers = {
+    "Accept": "application/json",
+    "Content-Type": "application/json",
+    "Authorization": "Bearer " + access_token,
+}
+transferResponse = requests.post(
+    "https://guc-spclient.spotify.com/connect-state/v1/connect/transfer/from/noop/to/{}".format(
```

Comment:
guc is for US.

It is not advisable to use spotify internal API:s. A simple force_playback with spotipy should suffice